code/controllers/Processes/ssd.dm

```diff
@@ -0,0 +1,57 @@
+
+var/global/datum/controller/process/ssd/ssd
+
+/datum/controller/process/ssd
+	var/list/current_ssds = list()
+	var/list/previous_ssds = list()
+	var/list/ignored_ssds = list()
+	var/list/areas_to_ignore = list(/area/security/prison)
+
+/datum/controller/process/ssd/setup()
+	name = "SSD"
+	schedule_interval = 6000
+
+/datum/controller/process/ssd/doWork()
+	current_ssds = list()
+	for(var/mob/living/carbon/human/H in mob_list)
```

Comment:
why loop through the entire mob list if the dead SSDs return 0 and aren't elligible for job slot clearing?